.circleci/cloud_nuke_config.yml

```diff
@@ -0,0 +1,19 @@
+s3:
```

Comment:
NIT: we should probably make it clearer what this file is. At the least, add a comment explaining that this is a config file specifically for the Gruntwork Phx DevOps accounts, where we want to delete certain S3 buckets, but not all. Might be worth renaming the file to make that clearer too.

Reply:
Also, I'm having a brain fart... But with the config file, it still deletes all resource types, and the config here just tells it how to filter `s3`, with the assumption all other resources are unfiltered? That is, by only specifying `s3` here, it doesn't mean _only_ delete S3, right?

Reply:
> we should probably make it clearer what this file is.

done: 447d363

> But with the config file, it still deletes all resource types, and the config here just tells it how to filter s3, with the assumption all other resources are unfiltered? That is, by only specifying s3 here, it doesn't mean only delete S3, right?

Yes, this config is just for specifying what s3 buckets to delete, not "just delete s3". Here is a part of the run log on my local machine that confirms this:

``` go run main.go aws \
              --older-than 1h \
              --config ./.circleci/cloud_nuke_config.yml

INFO[2020-10-15T09:20:30-05:00] The following resource types will be nuked:
INFO[2020-10-15T09:20:30-05:00] - ami
INFO[2020-10-15T09:20:30-05:00] - asg
INFO[2020-10-15T09:20:30-05:00] - ebs
INFO[2020-10-15T09:20:30-05:00] - ec2
INFO[2020-10-15T09:20:30-05:00] - ecscluster
INFO[2020-10-15T09:20:30-05:00] - ecsserv
INFO[2020-10-15T09:20:30-05:00] - eip
INFO[2020-10-15T09:20:30-05:00] - ekscluster
INFO[2020-10-15T09:20:30-05:00] - elb
INFO[2020-10-15T09:20:30-05:00] - elbv2
INFO[2020-10-15T09:20:30-05:00] - lc
INFO[2020-10-15T09:20:30-05:00] - rds
INFO[2020-10-15T09:20:30-05:00] - s3
INFO[2020-10-15T09:20:30-05:00] - snap
INFO[2020-10-15T09:20:32-05:00] Retrieving active AWS resources in [eu-north-1, ap-south-1, eu-west-3, eu-west-2, eu-west-1, ap-northeast-2, ap-northeast-1, sa-east-1, ca-central-1, ap-southeast-1, ap-southeast-2, eu-central-1, us-east-1, us-east-2,
```

Reply:
Roger, thx